github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -502,6 +502,90 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- name: periodic-kubevirt-e2e-k8s-latest-stable
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "0 22,10 * * *"
+  cluster: prow-workloads
+  decorate: true
+  decoration_config:
+    timeout: 7h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "false"
```

Comment:
There is no docker mirror yet?

Reply:
nope, we don't have yet dynamically provisioned persistent volumes in the new cluster

Reply:
I think it is critical to have a dedicated cache-node (otherwise the caches interfere with workloads, we can not mix them), like i the current cluster. You don't need PVCs for that. For caches it is even preferable to use host paths.

Reply:
On e.g. managed clusters partition layouts can not be influenced by us, disks size may not be big enough, and one can normally request fast storage, or storage which has independent networking, which makes PVCs a good option there too.

Reply:
Ok sounds good to me to have a dedicated cache node. We only have the new SR-IOV machines in this cluster, I think it would be better to use more regular bare metals as the cache node, maybe even the same srv05 we use on phx-prow, which already has in place the partitions, wdyt?

I still think we would need some kind of dynamically/statically provisioned PVs in this cluster, besides the cache workloads. prometheus will need them, and maybe other workloads too. I opened a support ticket in IBM UI and they proposed a few options, all of them from third-parties or ibm cloud operator, which I think would not work for our use case.

Reply:
> I still think we would need some kind of dynamically/statically provisioned PVs in this cluster, besides the cache workloads. prometheus will need them, and maybe other workloads too. I opened a support ticket in IBM UI and they proposed a few options, all of them from third-parties or ibm cloud operator, which I think would not work for our use case.

I would suggest to follow an architecture where prometheus in this setup is non-persistent, but it is scraped from another prometheus in a managed cluster with persistent storage. The prometheus in the managed cluster would then also fuel all dashboards.

Reply:
Indeed, i was thinking about using thanos for persisting and centralizing the metrics, but in any case I think we would need a small temporary storage for each of the prometheus instances that are going to be aggregated. Will check in any case.